content/projects/first-personal-website/_index.md

```diff
@@ -2,19 +2,19 @@
 _db_id: 220
 content_type: project
 flavours:
-- none
+  - none
```

Comment:
Why are you editing the spacing?

Reply:
All I did was just add the additional link at the bottom and it seems the added spaces automatically occurred in each line. I will rectify that as well as the merge conflicts.